packages/react/src/components/Button/Button.tsx

```diff
@@ -77,6 +77,9 @@ export interface ButtonProps
   /** A button can emphasize that it represents an alternative action. */
   secondary?: boolean
 
+  /** A button that represents an alternative action and also has contrast against dark backgrounds */
+  secondaryAlt?: boolean
```

Comment:
Why not just reusing the `secondary` prop? In my opinion it would be a good use-case. You can see different implementations here: https://open-ui.org/components/button Also, please update the changelog to reflect the real implementation.

Reply:
If we have like a 'transparent' property (or 'ghost', but I really dislike that term), it would only apply to a secondary button and that doesn't seem as intuitive to me. I think it's more straight-forward to use a new button variant here in the same way we don't reuse the primary button for secondary. What do you think?

Reply:
We don't want to just invent new concept and add it as a prop on Fluent, if it is not a common thing on the web. This is why I shared the link with you, so you can take a look on the common Button props that exists. Semantic has the prop basic: https://react.semantic-ui.com/elements/button/ for describing something similar. You may consider using this.